Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/actions/index.ts

```diff
@@ -41,4 +43,13 @@ const formatNotificationDate = (notificationDate: string) => {
         notificationDate = notificationDate.replace(',', '\xa0\xa0');
     }
     return notificationDate;
+}
+
+const formatNotificationSendingDuration = (SendingStartedDate: string, SentDate: string) => {
```

Comment:
Lowercase sendingStartedDate and sentDate parameters.

Reply:
done